modules/abstract-eth/src/lib/transactionBuilder.ts

```diff
@@ -824,7 +824,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
    * @param {number} version wallet version
    */
   walletVersion(version: number): void {
-    if (version < 0 || version > 4 || version === 3) {
+    if (version < 0 || version > 5 || version === 3) {
```

Comment:
It is throwing an error in case of Mpc wallet, i think this should be required only in the case of a wallet contract. Why there is a need of calling this with Mpcv2?

Reply:
forwarder deployment is failing, not quite sure why we are passing walletVersion while building deploy forwarder txn